x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/analysis/PreAnalyzer.java

```diff
@@ -17,9 +18,9 @@
 public class PreAnalyzer {
 
     public LogicalPlan preAnalyze(LogicalPlan plan, IndexResolution indices) {
-        // wrap a potential index_not_found_exception with a VerificationException (expected by client)
         if (indices.isValid() == false) {
```

Comment:
A comment is useful to indicate why the Verification exception is wrapped inside an infe